pkgs/development/python-modules/generic/default.nix

```diff
@@ -57,8 +59,37 @@ let
   # use setuptools shim (so that setuptools is imported before distutils)
   # pip does the same thing: https://github.com/pypa/pip/pull/3265
   setuppy = ./run_setup.py;
-  # For backwards compatibility, let's use an alias
-  doInstallCheck = doCheck;
+
+  # We run all tests after software has been installed since that is
+  # a common idiom in Python
+  # We therefore use doInstallCheck
+
+  formatspecific =
+    if format=="wheel" then
+    # Attributes specific for format="wheel"
+      {
+      unpackPhase = ''
+        mkdir dist
+        cp $src dist/"''${src#*-}"
+      '';
+      buildPhase = "";
```

Comment:
Can you add a comment explaining these options? For example, why there would be no `buildPhase`, and why would the tests be skipped?